source/dyaml/emitter.d

```diff
@@ -1004,7 +1004,7 @@ struct Emitter
         }
 
         ///Analyze specifed scalar and return the analysis result.
-        static ScalarAnalysis analyzeScalar(string scalar) @safe
+        static ScalarAnalysis analyzeScalar(string scalar)
```

Comment:
Just make this a template without arguments - `analyzeScalar()(string scalar)` as this will then also forward other attributes